vendor/knative.dev/reconciler-test/pkg/images/ko.go

```diff
@@ -16,7 +16,20 @@ limitations under the License.
 
 package images
 
+import (
+	"fmt"
+	"os"
+)
+
 // Use ko to publish the image.
 func KoPublish(path string) (string, error) {
-	return "", nil
+	platform := os.Getenv("PLATFORM")
+	if len(platform) > 0 {
+		platform = " --platform=" + platform
+	}
+	out, err := runCmd(fmt.Sprintf("ko publish%s -B %s", platform, path))
+	if err != nil {
+		return "", err
+	}
```

Comment:
this is not what we want :-)

Reply:
this is yet another symptom showing the weakness of the `rekt testing` framework - totally relying on `ko` (we can not rely (or even use `ko`) on our midstream, hence the hackery here)